assets/css/layout/_main.scss

```diff
@@ -0,0 +1,14 @@
+.main {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 75ch;
+  // padding: issues
+  padding: 0 27px;
+}
+
+@media screen and (min-width: 600px) {
+  .main {
+    max-width: 130ch;
```

Comment:
`130ch` for the main content width on the desktop might be too small. The picture below shows the 2 columns of articles have already stretched out of this width.

@lliskovoi might have insight.

<img width="1258" alt="Screen Shot 2020-03-24 at 4 24 56 PM" src="https://user-images.githubusercontent.com/572877/77473988-ba16fe80-6dec-11ea-9035-6d03e9c373d1.png">

Reply:
What value should I increase it too?